ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.jsonschema.json

```diff
@@ -14382,7 +14382,7 @@
           },
           "ipv6_address_virtual": {
```

Comment:
so it looks like we don't have the deprecation information in the json schema - is it on purpose?

Reply:
It is not. It would be good to add here to the description, and also do something on the removed keys. I have not looked at this yet.

Reply:
Added to jsonschema